tests/console/network.pm

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check for network daemon
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base 'consoletest';
+use strict;
+use testapi;
+use utils;
+use version_utils;
+
+sub run {
+    select_console 'root-console';
```

Comment:
why not to use serial terminal here ?

Reply:
Why should I?

Reply:
Well - actually I am using the serial line for script return status.
Like any other `console` test is doing it.